tools/obscuroscan_v2/frontend/src/stores/counterStore.js

```diff
@@ -13,11 +14,16 @@ export const useCounterStore = defineStore({
         async fetchCount() {
             this.loading = true;
             try {
-                console.log(Config.backendServerAddress)
-                const response = await fetch( Config.backendServerAddress+'/count/contracts/');
-                const data = await response.json();
-                this.count = data.count;
-                console.log("Fetched "+this.count);
+                let response = await fetch( Config.backendServerAddress+'/count/contracts/');
+                let data = await response.json();
```

Comment:
Is it safe to have `response` and `data` be overwritten variables like this? I think general principle to avoid risk of weird bugs is just to use `const` unless you've got a good reason.

Reply:
Totaly right, fixed it 👍